srml/assets/README.adoc

```diff
@@ -0,0 +1,108 @@
+# Assets Module
+
+## Overview
+
+The assets module provides functionality for asset management of fungible asset classes with a fixed supply, including:
+
+* Asset Issuance
+* Asset Transfer
+* Asset Destruction
+
+The SRML Support Module implements the `Currency` trait abstraction over the SRML Assets Module's fungible assets system.
+
+### Types
+
+* `AssetId`: `u32`
+
+## Public Interface
+
+### Supported Origins
+
+**signed** - Used to issue, transfer, and destroy an asset holding.
+
+### Public Immutable functions (getters)
+
+#### `balance(id: AssetId, who: T::AccountId) -> T::Balance`
+
+#### `total_supply(id: AssetId) -> T::Balance`
+
+### Public Mutable functions (changing state)
+
+#### `issue(origin, total: T::Balance)`
+
+#### `transfer(origin, id: AssetId, target: <T::Lookup as StaticLookup>::Source, amount: T::Balance)`
+
+#### `destroy(origin, id: AssetId)`
+
+### Events:
+
+#### `Issued(AssetId, AccountId, Balance)`
+
+#### `Transferred(AssetId, AccountId, AccountId, Balance)`
+
+#### `Destroyed(AssetId, AccountId, Balance)`
+
+## Usage
+
+The following example shows how to use the Asset Module in your custom module to query an account's asset holding, and query the total supply of an asset and an account's asset holding balance.
+
+**1. Import Asset Module and Types**
+
+Import the `assets` module and derive your custom module configuration traits from the `assets` module trait.
+
+```rust
+use assets;
+
+pub trait Trait: assets::Trait { }
+```
+
+**2. Import Accounts**
+
+```rust
+const ALICE: u64 = 1;
+const BOB: u64 = 2;
+
+// Initial Asset ID is 0
+let initial_asset_id: u32 = <assets::Module<T>>::next_asset_id();
```

Comment:
`initial_asset_id`? why the initial here? should just call it `asset_id`, and it can be other number if this is not the very first asset issued.